packages/block-library/src/preformatted/block.json

```
@@ -12,6 +12,7 @@
 		}
 	},
 	"supports": {
-		"anchor": true
+		"anchor": true,
+		"fontSize": true
 	}
```

In https://github.com/WordPress/gutenberg/pull/25220 which was merged a few days ago, we allow block-styles to only be loaded when a block actually exists on a page for FSE themes. As a result of that change, when adding a stylesheet to a block we need to declare it in the `block.json` file.
Adding this line will allow FSE themes to load the stylesheet :+1: 
```suggestion
	},
	"style": "wp-block-preformatted"
```